src/test/java/stepDefinitions/StepsArticles.java

```diff
@@ -87,4 +96,69 @@ public void iClickTheDeleteButton() {
     public void theArticleWasSuccessfullyRemoved() {
         Assert.assertEquals(ARTICLE_DELETE_ERROR, ARTICLES_DELETE_MSG, articleReadPage.getAlertMsg());
     }
+
+    @And("the system displays the list of items")
+    public void theSystemDisplaysTheListOfItems() {
+        Assert.assertTrue(articlesFeedPage.articlesTitles().size() > 0);
+        Articles articles = new Articles();
+        articles.setTitleArticle(randomText(articlesFeedPage.articlesTitles()));
+        DataManager.getInstance().setArticles(articles);
+
+    }
+
+    @And("click on an article title in the list")
+    public void clickOnAnArticleTitleInTheList() {
+        articlesFeedPage.clickSpecificTitleArticle(DataManager.getInstance().getArticles().getTitleArticle());
+    }
+
+    @And("the system opens the article information")
+    public void theSystemOpensTheArticleInformation() {
+        Assert.assertEquals(ARTICLE_TITLE_ERROR, DataManager.getInstance().getArticles().getTitleArticle(), articleReadPage.getTitleArticle());
+    }
+
+    private String randomText(List<String> text) {
+        Random random = new Random();
+        return text.get(random.nextInt(text.size()));
```

Comment:
Does this work as expected?

Reply:
yes, that method receives a list of strings and returns a random string from that list

Reply:
Nice!